templates/stripes/stripe-infographic.php

```diff
@@ -1,25 +1,36 @@
-<?php if (!empty($image)) : ?>
-<div class="<?= $this->e($type, 'wrapperClasses') ?>"<?php
-    echo !isset($is_enlargeable) || $is_enlargeable ? ' data-is-enlargeable="true"' : '';
-    echo !empty($variation) ? " data-variation=\"{$variation}\"" : '';
+<?php if (!empty($image)): ?>
+<div class="<?= $this->e($type, "wrapperClasses") ?>"<?php
+echo !isset($is_enlargeable) || $is_enlargeable
+  ? ' data-is-enlargeable="true"'
+  : "";
+echo !empty($variation) ? " data-variation=\"{$variation}\"" : "";
 ?>>
     <div class="vssl-stripe-column">
-        <?php if (!empty($title['html'])) : ?>
-        <div class="vssl-stripe--infographic--title"><?= $this->inline($title['html']) ?></div>
+        <?php if (!empty($title["html"])): ?>
+        <div class="vssl-stripe--infographic--title"><?= $this->inline(
+          $title["html"]
+        ) ?></div>
         <?php endif; ?>
 
         <div class="vssl-stripe--infographic--image">
-            <img src="<?= $this->image($image, !empty($image_style) ? $image_style : null) ?>"
-                alt="<?= $image ?>" loading="lazy" />
+            <img src="<?= $this->image(
+              $image,
+              !empty($image_style) ? $image_style : null
+            ) ?>"
+                alt="<?= $image_alt ?? ($image ?? "") ?>" loading="lazy" />
```

Comment:
Using new `$image_alt` property for the alt attribute, defaulting to the image path if not set